OpenRA.Mods.Common/Widgets/Logic/Ingame/GameInfoStatsLogic.cs

```diff
@@ -159,6 +177,12 @@ public GameInfoStatsLogic(Widget widget, World world, OrderManager orderManager,
 							confirmText: "Kick");
 					};
 
+					var muteCheckbox = item.Get<CheckboxWidget>("MUTE");
+					muteCheckbox.IsChecked = () => TextNotificationsManager.MutedPlayers[client.Index];
+					muteCheckbox.OnClick = () => TextNotificationsManager.MutedPlayers[client.Index] ^= true;
+					muteCheckbox.IsVisible = () => !client.IsBot && client.State != Session.ClientState.Disconnected && client.Index != orderManager.LocalClient?.Index;
+					muteCheckbox.GetTooltipText = () => muteCheckbox.IsChecked() ? "Unmute this player" : "Mute this player";
```

Comment:
This should use the translations I assume.

Reply:
not the first time I forgot the code is duplicated